packages/protocol/contracts/common/interfaces/IMintCeloSchedule.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
update `getTargetGoldTotalSupply` to `getTargetCeloTotalSupply`. Including the NATSPEC